program2.js

```diff
@@ -0,0 +1,8 @@
+var result = 0;
+var i;
+
+for (i = 2; i < process.argv.length; i++) {
```

Comment:
iはここで宣言したほうがスコープが小さくて良い

```suggestion
for (let i = 2; i < process.argv.length; i++) {
```